php-laravel-phpcs.md

```diff
@@ -0,0 +1,139 @@
+# Introduction
+
+[PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) is a set of two PHP scripts; the main phpcs script that tokenizes PHP, JavaScript and CSS files to detect violations of a defined coding standard, and a second phpcbf script to automatically correct coding standard violations. PHP_CodeSniffer is an essential development tool that ensures your code remains clean and consistent.
+
+# Prerequisite
```

Comment:
Can be Prerequisites
![image](https://user-images.githubusercontent.com/26721594/165812780-9b44fac5-4285-4335-a376-2ce1098aeabc.png)

Reply:
Done